templates/base.html

```diff
@@ -5,13 +5,17 @@
   <head>
     <title>{% block title %}PolarBear{% endblock %}</title>
     <link rel="stylesheet" href="{% static 'base.css' %}">
+    {% block stylesheets %}{% endblock %}
   </head>
   <body>
     <div class="mainbar">
       <h1><a href="/" class="nolinkstyle">PolarBear</a></h1>
       {% if user and user.is_authenticated %}
       <div class="baroption">
-        <p style = 'padding-right: 10pt'>welcome, {{ user.username }}</p>
+        <p>welcome, {{ user.username }}</p>
+        <a href="/user/authors/{{ user.author.pk }}" class="buttonlink">Profile</a>
```

Comment:
all urls should use django's reverse urls i.e. `{% url 'profile' user.author.pk %}`